cool-demos/slash-commands.html

```diff
@@ -0,0 +1,247 @@
+
+            // An array holding all unique details of each menu item in the
+            // slash commands menu which is later fed into the autocompleter API.
+            var insertActions = [
+                {
+                    text: 'Heading 1',
+                    icon: 'h1',
+                    action: () => {
+                        // Declare what content to be inserted, or what command to execute.
+                        // https://www.tiny.cloud/docs/api/tinymce/tinymce.editor/#insertcontent
+                        // https://www.tiny.cloud/docs/api/tinymce/tinymce.editor/#execcommand
+                        editor.insertContent('<h1>Heading 1</h1>');
+
+                        // Certain actions, like headings, inserts a placeholder to provide some user feedback
+                        // But since the placeholder is meant to be replaced
+                        // we can force TinyMCE to select the placeholder so the user can just begin
+                        // typing their heading
+                        // https://www.tiny.cloud/docs/api/tinymce.dom/tinymce.dom.selection/
+                        editor.selection.select(editor.selection.getNode());
+                    }
+                },
+                {
+                    text: 'Heading 2',
+                    icon: 'h2',
+                    action: () => {
+                        editor.insertContent('<h2>Heading 2</h2>');
+                        editor.selection.select(editor.selection.getNode());
+                    }
+                },
+                {
+                    text: 'Heading 3',
+                    icon: 'h3',
+                    action: () => {
+                        editor.insertContent('<h3>Heading 3</h3>');
+                        editor.selection.select(editor.selection.getNode());
+                    }
+                },
+                {
+                    text: 'Bulleted list',
+                    plugin: 'lists',
```

Comment:
Is this a left over from the hasPlugin filtering remove them all. Since this is a demo it makes less sense to make this smart like that.

Reply:
You mean the plugin property? Yeah, let's remove that.